utils/count-vendors.mjs

```diff
@@ -19,12 +19,7 @@ export default function countVendors(feature) {
 	return Object.keys(Engines).reduce((acc, engine) => {
 		const isSupportedOnEngine = Engines[engine].some(browser => {
 			const support = feature.browser_support[browser];
-			if (!browserVersionHasReleaseDate(browser, support)) {
-				return false;
-			}
-
-			// Do not allow pre-releases to count as a vendor implementation
-			return typeof support === 'string' && /^[0-9|.]+$/.test(support);
+			return browserVersionHasReleaseDate(browser, support);
```

Comment:
`browserVersionHasReleaseDate` also checks for pre-release versions, so this was duplicate logic.